io.github.lxi-tools.yaml

```diff
@@ -0,0 +1,142 @@
+---
+app-id: io.github.lxi-tools
```

Comment:
A github based app-id include the user/org name AND the app name. So it is made of 4 components.

Reply:
Right, I overlooked that. I did actually name it correctly in the appdata.xml file that my application installs. Fixed.